### PR TITLE
Updated slots, merging with latest master

### PR DESCRIPTION
**Pull request by TimmahC**
pls review and change the configs yourself. the emotes for slots should be universal, i also use the coin emote, so you'll have to change that. feel free to also change the payouts/probabilities.

### CODE REVIEW

File: chowder/chowder_config.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "guild_id": 454492770682404875,
+    "guild_id": 493580256079511575,
```
Comment (vsporeddy) [inline chowder/chowder_config.json:2]: Just omit these files from the commit pls

File: games/slots/slots_config.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      "RIP": {
+        "label": "Nothing", 
+        "payout": 0, 
+        "message": "```css\nYou hate to see that...```", 
+        "color": 10717577 
+      }
+    },
+    "rip_emote": "<:ChowderRIP:744794804948435014>",
+    "nbad_emote": "<:SpatikaClutch:609867523155951621>",
+    "coin_emote": "<:ChowderCoin:750409427483492501>"
```
Comment (vsporeddy) [inline games/slots/slots_config.json:70]: Change this to ChowderCoin:746800434873761955

File: slots_test.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
+from games.slots import slots
+import unittest
+
+class TestSlots(unittest.TestCase):
+
+
```
Comment (vsporeddy) [inline slots_test.py:6]: why these spaces

File: games/slots/slots_config.json (hunk 2 above)
Comment (vsporeddy) [reply games/slots/slots_config.json:70]: change the other ones too if they're localized

File: games/slots/slots.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+with open("games/slots/slots_config.json", "r") as read_file:
+    config = json.load(read_file)
+
+emotes = config["emotes"]
+reels = config["reels"]
+wildcard = config["wildcard"]
+symbols = config["symbols"]
+win_hands = config["winning_hands"]
+
+async def roll(self, ctx, bet):
```
Comment (vsporeddy) [inline games/slots/slots.py:18]: this function hurts to read but I'll just assume it works

File: config.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
-    "guild": "Dark Times", 
+    "guild": "chowderbot test",
```
Comment (vsporeddy) [inline config.json:2]: this one too

File: games/gamble/gamble.py [hunk 6]
```diff
@@ -28,6 +29,17 @@ async def roll(self, ctx, max_roll: int = 6):
     async def flip(self, ctx):
         await ctx.send(random.choice([config["heads"], config["tails"]]))
 
+    @commands.command(name="slots", brief="Try your hand at the slots and get rich.")
+    async def slots(self, ctx, *args):
+        if (len(args) == 0):
+            bet = None
+        elif (len(args) != 1):
```
Comment (vsporeddy) [inline games/gamble/gamble.py:36]: if we're only allowing one arg why not just have a named param

Comment (TimmahC) [reply games/gamble/gamble.py:36]: i think i did it this way just for that error msg. can prolly remove it.

**Pull request merged**